cdap-runtime-ext-dataproc/src/test/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClientTest.java

```diff
@@ -61,13 +63,10 @@ public void init() throws Exception {
     properties.put("accountKey", "{ \"type\": \"test\"}");
     properties.put(DataprocConf.PROJECT_ID_KEY, "dummy-project");
     properties.put("zone", "us-test1-c");
-    dataprocConf = DataprocConf.create(properties, null);
+    dataprocConf = DataprocConf.create(properties);
 
-    PowerMockito.spy(DataprocClient.class);
```

Comment:
Thanks! static method mocking is always a code smell

Reply:
👍  I didn't even know you could mock static methods until seeing this test